src/b-splines/indexing.jl

```diff
@@ -51,6 +51,11 @@ end
     getindex_impl(itp)
 end
 
+function (itp::BSplineInterpolation{T,N,TCoefs,IT,GT,pad})(args...) where {T,N,TCoefs,IT,GT,pad}
```

Comment:
sorry if I'm being dense, but do you really need to specify all the parameters? wouldn't

```
(itp::TheITPType)(args...) = itp[args...]
```
work?